.github/renovate/semanticCommits.json5

```diff
@@ -1,115 +1,97 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "packageRules": [
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
Same goes for this file: I think we should retain the quotes on the renovate config due to `Property keys must be doublequoted [jsonc]` errors in vscode.